### PR TITLE
lowercase-plugin

### PR DESCRIPTION
**Pull request by thev19**
**Lowercase-plugin** 
- Added a plugin to check if the given string is lowercase or not.

**Related Issues:** 
- #514

### CODE REVIEW

**Comment by b4s36t4** [comment]
@thev19 PR seems to be having some conflicts, can you resolve them first?

**Comment by thev19** [comment]
@b4s36t4 Resolved the merge conflicts.

File: plugins/default/manifest.json [hunk 1]
```diff
@@ -462,6 +462,19 @@
...
+      "type": "guardrail",
+      "supportedHooks": ["beforeRequest", "afterRequest"],
+      "description": [
+        {
+          "type": "subHeading",
+          "text": "Checks if content has all lowercase letters."
+        }
+      ],
+      "parameters": {}
+    }
```
Comment (b4s36t4) [inline plugins/default/manifest.json]: This seems to be conflict issue, can you fix this?

File: plugins/default/default.test.ts [hunk 2]
```diff
@@ -754,3 +755,28 @@ describe('endsWith handler', () => {
     expect(result.data).toBe(null);
   });
 });
+
+describe('allLowercase handler', () => {
```
Comment (b4s36t4) [inline plugins/default/default.test.ts:759]: Test cases are only added for `afterRequestHook`. Can we please add the same for `beforeRequestHook` as well please?

**Review by b4s36t4 — CHANGES_REQUESTED** [review]

File: plugins/default/default.test.ts (hunk 2 above)
Comment (thev19) [reply plugins/default/default.test.ts:759]: added beforeRequestHook.

**Review by b4s36t4 — APPROVED** [review]

**Pull request merged**